newIDE/app/src/PropertiesEditor/index.js

```diff
@@ -206,7 +206,7 @@ export default class PropertiesEditor extends React.Component<Props, {||}> {
         key={field.name}
         fullWidth
         primary
-        label={field.getLabel(this.props.instances[0])}
```

Comment:
There is already a "getLabel" in InstancePropertiesEditor, so we should not remove the usage of getLabel.

Reply:
Oooh, I didn't find the "getLabel" method in the PropertiesMapToSchema, so I thought the button code was old and broken... where is the getLabel definition?

Reply:
I see, there's a getLabel but defined in the instances properties only... so buttons won't work for generalized behavior properties, I think I tried to render buttons and it always crashed, hence the idea the code was outdated. I get it now :)